lib/beaker/hypervisor/vagrant.rb

```diff
@@ -106,7 +106,7 @@ def get_ip_from_vagrant_file(hostname)
       ip = ''
       if File.file?(@vagrant_file) #we should have a vagrant file available to us for reading
         f = File.read(@vagrant_file)
-        m = /#{hostname}.*?ip:\s*('|")\s*([^'"]+)('|")/m.match(f)
+        m = /'#{hostname}'.*?ip:\s*('|")\s*([^'"]+)('|")/m.match(f)
```

Comment:
well currently beaker always adds the ' mark anyways, so the above regex will always match the correct hostname's IP.